Forc test change (#297)
FuelLabs/sway-applications: .docs/contributing-book/src/documentation/project-quality/testing.md

### `harness.rs`

> This file is the entry point for the tests, and thus it contains only the modules `functions` and `utils`. This is what is executed when `cargo test` is run.

Reviewer: ```suggestion
The `harness` file is the entry point for the tests, and thus it contains the `functions` and `utils` modules. This is what is executed when `cargo test` is run.
```

> This file is the one that is called by `forc build && cargo test` and thus it only contains the modules `functions` and `utils` in order to bring them into scope so that the tests can run.

Reviewer 2: ```suggestion
This file is the entry point for the tests, and thus it contains only the modules `functions` and `utils`. This is what is executed when `cargo test` is run.
```

> This file is the one that is called by `forc build && cargo test` and thus it only contains the modules `functions` and `utils` in order to bring them into scope so that the tests can run.

Reviewer 2: ```suggestion
This file is the entry point for the tests, and thus it contains only the modules `functions` and `utils`. This is what is executed when `cargo test` is run.
```
